src/book.py

```diff
@@ -363,6 +363,123 @@ def _read_kraken_ledgers_old(self, file_path: Path) -> None:
 
         self._read_kraken_ledgers(file_path)
 
+    def _read_bitpanda_pro_trades(self, file_path: Path) -> None:
+        """
+        Read trade statement from Bitpanda Pro.
+        """
+
+        platform = "bitpanda_pro"
+        with open(file_path, encoding="utf8") as f:
+            reader = csv.reader(f)
+
+            # should be true in any case, since this has been dispatched here
+            # so simply skip
+            line = next(reader)
+            # assert line[0] == \
+            # "Disclaimer: ..."
+
+            # for transactions, it's currently written "id" (small)
+            line = next(reader)
+            if line[0].startswith("Account id :"):
+                log.warning(
+                    f"{file_path} looks like a Bitpanda transaction file. Skipping."
```

Comment:
You might want to extend the warning and explain how the user can fix this. I guess he/she has to download die correct file?